data/MBS/data.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "MBS",
```

Comment:
Should this be "MonkeyLeague" instead of the token symbol?

Reply:
This is typically set to the name, i.e. MonkeyLeague

Reply:
it's by design, we renamed it to `MBS`(from  `MonkeyLeague`)